pytorch_lightning/loggers/tensorboard.py

```diff
@@ -100,6 +103,7 @@ def __init__(
         self._default_hp_metric = default_hp_metric
         self._prefix = prefix
         self._fs = get_filesystem(save_dir)
+        self._flush_logs_every_n_steps = flush_logs_every_n_steps
```

Comment:
@awaelchli @edward-io isn't this redundant with `max_queue` and `flush_secs` in the tensorboard summary writer constructor? https://pytorch.org/docs/stable/_modules/torch/utils/tensorboard/writer.html#SummaryWriter

Reply:
It is redundant (now we have three different ways to flush logs!). It depends how strongly if we want to maintain backwards compatibility by keeping the old behavior of flushing per step.

Reply:
IMO it's better to describe how someone can configure the existing tensorboard logger for this. Given kwargs are passed through from the constructor to the experiment writer, we could demonstrate how max queue and flush_secs could be used to configure this behavior. I prefer relying on the libraries existing APIs rather than establishing something on top that's lightning specific especially because all these loggers have different APIs .

@awaelchli what do you prefer?

Reply:
ok I guess for the constructor that's fair.